public/release_action.js

```diff
@@ -30,8 +30,6 @@ function hex2int(hex) {
 }
 
 $(function() {
-	socket.emit('release_actions_get');
```

Comment:
Are these sent somewhere else?

Reply:
Technically this should be be a `socket.emit('get_actions');` but its hooking by luck the same call from `public/actions.js`.